.github/workflows/cargo-publish.yml

```diff
@@ -33,9 +33,16 @@ jobs:
       # The script will retry publish for 5 times with 60 seconds between the retries.
       - name: Public mmtk-core
         run: |
+          success=false
+
           for n in {1..5}; do
-            echo "Attempt #"$n
-            cargo publish && break
-            echo "Wait for Retry #"$n
-            sleep 60
+              echo "Attempt #"$n
+              false && { success=true; break; }
```

Comment:
`false` is probably from testing but it should be `cargo publish`

Reply:
Right. Sorry for that.